label_studio/projects/functions/utils.py

```diff
@@ -26,3 +26,17 @@ def make_queryset_from_iterable(tasks_list):
                 raise ValueError(f'Unknown object type: {str(task)}')
         queryset = Task.objects.filter(id__in=ids)
     return queryset
+
+
+def recalculate_created_labels_from_scratch(project, summary):
+    summary.created_labels = {}
+    if project.annotations.exists():
+        summary.update_created_annotations_and_labels(project.annotations.all())
+    else:
+        summary.save(update_fields=['created_labels'])
```

Comment:
ditto feedback below re: the branching, though it's more complicated in this case because there are two fields potentially affected by the first codepath.

Reply:
should be fixed now